pop-cli-for-appchains/pop-cli/call.md

```diff
@@ -0,0 +1,74 @@
+---
+description: Call a chain
+---
+
+# call
+
+```bash
+pop call <COMMAND>
+```
+
+**Interacting with the chain**
+
+```bash
+pop call chain
+```
+
+You will be prompted to select a pallet, choose a dispatchable function, specify any required arguments, then provide connection details and the information of the account that will sign the extrinsic.
+
+If interactive guidance is not desired, you can proceed manually as follows:
+
+```bash
+pop call chain --pallet System --function remark --args "0x11" --url ws://localhost:9944/ --suri //Alice
+```
+
+If you have sudo privileges and need to dispatch a function as Root origin, include the `--sudo` flag in your command.
```

Comment:
As my comment above, would move away from saying "having sudo privileges".

Reply:
As above. Solve it in https://github.com/r0gue-io/pop-docs/pull/12/commits/1090c54a67f8a58d84e3b649f99877b70aa9aeaa